azure-ipam/main.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"os"
+
+	cnsclient "github.com/Azure/azure-container-networking/cns/client"
+	"github.com/containernetworking/cni/pkg/skel"
+	"github.com/containernetworking/cni/pkg/version"
+	bv "github.com/containernetworking/plugins/pkg/utils/buildversion"
+	"github.com/pkg/errors"
+	"go.uber.org/zap"
+)
+
+func main() {
+	if err := executePlugin(); err != nil {
+		log.Printf("error executing azure-ipam plugin: %v\n", err)
+		os.Exit(1)
+	}
+}
+
+func executePlugin() error {
+	zapConfig := []byte(`{
+		"level": "debug",
+		"encoding": "json",
+		"outputPaths": ["/tmp/logs"],
```

Comment:
This might be an okay default, which we usually expect to get overridden by an explicit config option. I think this yaml []byte is unnecessary and we should be putting the logger/config together directly, though.